apps/web/src/screens/Dashboard/index.tsx

```diff
@@ -28,6 +29,17 @@ export const DashboardScreen: FC = () => {
 		await router.navigate('/');
 	};
 
+	const setLatestPathname = async () => {
+		if (pathname !== setting?.currentScreen) {
+			console.log(setting?.currentScreen);
+			await setPathname({ currentScreen: pathname });
+		}
+	};
+
+	useEffect(() => {
```

Comment:
this observe/watch flow should be in higher level, e.g App to ensure it work for other routes as well